test/message/source_map_throw_icu.out

```diff
@@ -1,6 +1,6 @@
 *icu.jsx:3
     ("********", throw Error("an error"));
-                       ^
+     ^
```

Comment:
Are we able to move the pointer to where `throw` is?

Reply:
Actually, this is where v8 suggested. Changes need to be made in v8.